index.html

```diff
@@ -1,15 +1,38 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Valera Mykytenko - CV</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Valera Mykytenko - CV</title>
 </head>
-
 <body>
-
+   <div><h1>Valera Mykytenko</h1></div>
+   <div><h2>Job Title</h2></div>
```

Comment:
```suggestion
   <h1>Valera Mykytenko</h1>
   <h2>Job Title</h2>
```
Ось ці теги видалити, я мав на увазі)